src/main/resources/application-dev.yml

```
@@ -0,0 +1,46 @@
+    redis:
+      host: redis://redis-container:6379
+
+logging:
+  level:
+    root: info
+    org.springframework.jdbc.core.JdbcTemplate: debug
+    org.springframework.jdbc.core.StatementCreatorUtils: trace
+
+jwt:
+  secret:
+    key: 60066e95d82266c1ae7bc5bf01828f64f4337ee0f4b4a1f29008ba63d7b3e69fb77d2abca7f2085ca7d6a8fabd224007cfd3316d0b96f714ea4215dd06988c59
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)